source/features/unreleased-commits.tsx

```diff
@@ -97,6 +122,7 @@ async function init(signal: AbortSignal): Promise<false | void> {
 	await api.expectToken();
 
 	observe(branchSelectorParent, add, {signal});
+	observe(branchSelector, addForNewRepositoryOverview, {signal});
```

Comment:
When adding code that supports multiple versions:

- add a TODO comment with a date 6-8 months from now to remind us to delete the old code
- make it easy to delete the old code without having to rewrite much else
- don't use "new" in the names; the "new" code is actually just "current" in the future